samples/bookinfo/src/reviews/Dockerfile

```diff
@@ -12,13 +12,21 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+FROM gradle:4.8.1 as builder
+
+# Not sure why but we need root to build
+USER 0
```

Comment:
Is it because we are trying to use /home/gradle?  @jwendell had some User changes and not sure if this would cause issues with those or not. I know some of the Dockerfile's have User 1000.

Reply:
/home/gradle should be owned by `1000` I think. Anyhow, we currently do `docker run --user 0` anyways so this is the same